### DISCUSSION TITLE
Ownership of twitter/websites/images

### DISCUSSION THREAD

**Discussion by glennawatson** [dotnet-foundation/Home#51] [General Discussions]
There have been other discussions associated with NuGet and GitHub

Questions have also been raised who owns the websites, social media accounts and images associated with the projects?

**Comment by rprouse** [comment]
We need to look into the agreements that we signed and I think it is different under assignment vs contribution but there are sections about trademarks and patents. For example, I believe NUnit agreed to allow the Foundation to use our logos, docs, etc, but didn't transfer our copyrights. I also know that various project have side agreements that weren't well tracked or transitioned between directors.

The board talked very briefly about this. I think we need to work with all of our member projects to understand every agreement and build a database so that it is clear to everyone and it reflects the project maintainers wishes. Some projects want to assign all the rights over to the foundation to ensure continuity of the project or for other reasons but others prefer to keep those rights in their control.

EDIT: We may also find that it is not well defined in the agreements that we signed. If that is the case, then I think we need to work with projects to settle this.

**Reply by jglozano** [reply]
@rprouse like always, great points. I would add that we might need a Standard Operating Procedures (SOPs) for projects as well. Not to add bureaucracy but to help with continuation.

Since the board members might change faster than project maintainers, it is very important to know context and have a history of what happened when/how/who. This along with the agreements will help future board members and maintainers.

**Reply by mitchelsellers** [reply]
I can confirm for my project there is no provision or governance or anything that is not the GitHub repo. 

And there should most likely be.

**Comment by BillWagner** [comment]
The master agreement does not mention twitter, websites, logos or other marks. See https://github.com/dotnet-foundation/Home/discussions/47#discussioncomment-1455596
